src/main/java/org/mabb/fontverter/cff/CffFontAdapter.java

```diff
@@ -196,16 +196,20 @@ public int getMaxY() {
     }
 
     public Encoding getEncoding() {
-        if (font instanceof EncodedFont) {
-            try {
-                return ((EncodedFont) font).getEncoding();
-            } catch (IOException e) {
-                return CFFStandardEncoding.getInstance();
-            }
-        }
+		if (font instanceof EncodedFont) {
+			try {
+				Encoding encoding = ((EncodedFont) font).getEncoding();
+				if (encoding.getCodeToNameMap().values().size() > 1) {
+					return encoding;
+				}
+			} catch (IOException e) {
+				e.printStackTrace();
```

Comment:
Change this to a logger stack trace error log not a print stack trace, search in files for exceptionutils or something to see an example

Reply:
ok, I will fix it.